godtools/Views/Cells/LanguageTableViewCell.swift

```diff
@@ -39,15 +39,9 @@ class LanguageTableViewCell: UITableViewCell {
         self.selectedBackgroundView = selectedView
     }
     
-    func languageExist(exist:Bool) {
-        if exist {
-            self.deleteButton.isHidden = false
-            self.downloadButton.isHidden = true
-        }
-        else {
-            self.deleteButton.isHidden = true
-            self.downloadButton.isHidden = false
-        }
+    func languageExist(_ exist:Bool) {
```

Comment:
This is very small, but could you change "Exist" to "Exists" and "exist" to "exists" please? It will just read a little bit better. Then this is 👍